src/sim.py

```diff
@@ -125,29 +143,47 @@ def simulate(
   dispatch.run_model(model_info, no_charger_scenario)
   tripsdf = ConvertVectorOfStructsToDataFrame(no_charger_scenario.trips)
   ncbuses = dispatch.count_buses(no_charger_scenario.trips)
+  nccost = no_charger_scenario.cost
```

Comment:
Break apart simulations into separate functions, like we discussed.

Reply:
I'd like to do this in a separate PR.

Reply:
Okay.